bin/main.rb

```diff
@@ -1 +1,2 @@
-puts "Hello world!"
+#!/usr/bin/env ruby
+puts '"Hello world!"'
```

Comment:
## Leave simple quotes
Change from `puts '"Hello world!"'` to `puts 'Hello world!'`